src/RedisRateLimiting/Concurrency/RedisConcurrencyRateLimiter.cs

```diff
@@ -28,7 +26,7 @@ public class RedisConcurrencyRateLimiter<TKey> : RateLimiter
 
         public override TimeSpan? IdleDuration => Interlocked.CompareExchange(ref _activeRequestsCount, 0, 0) > 0
             ? null
-            : new TimeSpan((long)((Stopwatch.GetTimestamp() - _idleSince) * TickFrequency));
+            : Stopwatch.GetElapsedTime(_idleSince);
```

Comment:
Can you please update the other rate limiters as well?

Reply:
updated